M2/Macaulay2/packages/Macaulay2Doc/monomorderings.m2

```diff
@@ -24,6 +29,9 @@ document {
      "Obtain the variables (generators) of the ring by subscripting the name of 
      the ring.  As always in Macaulay2, indexing starts at 0.",
      EXAMPLE "R_0^10+R_1^3+R_2",
+     "It is also possible to obtain the variables in a ring from strings
+     containing their names.",
+     EXAMPLE ///R_"a"^10+R_"b"^3+R_"c"///,
```

Comment:
Thanks for adding all this!

Side note: this notation is actually deprecated in favor of `String_Ring` like `"a"_R`, because it's more consistent with things like `1_R` and also promotion using `f_R`.

Reply:
To be clear, I don't remember who exactly decided it should be deprecated. If it was me, I'm fine with keeping it for backwards compatibility, but I think having both can be confusing.

Reply:
Ok -- I'll switch the example to use `"a"_R`, etc.